docs/hub-tutorials/join-mainnet.md

```diff
@@ -178,7 +177,7 @@ Check that everything is running smoothly:
 gaiad status
 ```
 
-View the status of the network with the [Cosmos Explorer](https://cosmos.network/launch).
+View the status of the network with the [Cosmos Explorer](https://www.mintscan.io/cosmos).
 
```

Comment:
Would suggest removing this line as it was just intended for when Cosmos launched.

```suggestion
```